src/WhatToCook.Application/Domain/Recipe.cs

```diff
@@ -6,23 +6,26 @@ public class Recipe
     public string Name { get; private set; }
     public string Description { get; private set; }
     public string TimeToPrepare { get; private set; }
-    public List<Ingredient> Ingredients { get; set; } = new();
+    public List<Ingredient> Ingredients { get; private set; } = new();
     public Statistics Statistics { get; set; }
     public string Image { get; private set; }
-    public List<PlanOfMeals> PlansOfMeals { get; set; }
-    public Recipe(string name, string description, string timeToPrepare, List<Ingredient> ingredients, Statistics statistics, string image, List<PlanOfMeals> plansOfMeals)
+    public List<RecipePlanOfMeals> RecipePlanOfMeals { get; private set; } = new();
+
+    public Recipe(string name, string description, string timeToPrepare, List<Ingredient> ingredients, Statistics statistics, string image)
     {
         SetName(name);
         SetDescription(description);
         SetTimeToPrepare(timeToPrepare);
         Ingredients = ingredients;
         Statistics = statistics;
```

Comment:
I think when we create the recipe, statistics can be created along with it because it will have default values - all will be 0

Reply:
done, will be in the next commit